src/neo/Ledger/Blockchain.cs

```diff
@@ -310,6 +310,7 @@ private VerifyResult OnNewExtensiblePayload(ExtensiblePayload payload)
         {
             DataCache snapshot = system.StoreView;
             extensibleWitnessWhiteList ??= UpdateExtensibleWitnessWhiteList(system.Settings, snapshot);
+            if (system.RelayCache.Any(u => u is ExtensiblePayload ext && ext.Sender == payload.Sender)) return VerifyResult.PolicyFail;
```

Comment:
Is it better to replace the old one? Otherwise the new one will never be allowed.

Reply:
I don't think we need to care about this. Because `RelayCache` will automatically remove the old items.

Reply:
> Because RelayCache will automatically remove the old items.

That's a part of the problem. Anyone can send 100 messages and kick out everything from `RelayCache`. I think we should limit the number of messages from one sender in the cache, just not to one message per sender, some 10% of the cache seem to be a reasonable limit.

Reply:
The attacker must be a special role on the blockchain. And his signature is in `ExtensiblePayload`. The committee can remove him easily.

Reply:
> The committee can remove him easily.

If transaction is accepted. But if we're talking about real attack there's gonna be lots of `Extensible` payloads, so that consensus messages won't be delivered properly and blocks can be delayed for a long time.

Reply:
Maybe we should replace the old one.